src/button/notification.scss

```diff
@@ -0,0 +1,58 @@
+// Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+// See LICENSE.md in the project root for license terms and full copyright notice.
+@import '../style/index';
+
+@mixin iui-button-notification($status: primary) {
+  position: relative;
+
+  &::before {
+    content: ' ';
+    position: absolute;
+    width: $iui-s;
+    height: $iui-s;
+    border-radius: 100%;
+    @include themed {
+      background-color: t(iui-color-foreground-#{$status});
+      border: 2px solid t(iui-color-background-1);
+    }
+    @media (prefers-reduced-motion: no-preference) {
+      transition: border-color $iui-speed-fast ease-out;
+    }
+  }
+
+  &:enabled:hover::before,
```

Comment:
What it does?

Reply:
Without this, on hover of the button the border of the notification dot does not match the button background color. Screenshot without that CSS: ![Screen Shot 2021-09-20 at 3 35 05 PM](https://user-images.githubusercontent.com/849817/134063370-0093da3a-e357-4448-88b2-f150efb9df34.png)

Reply:
Hm, so notification marker can only be used on borderless button? Or can we use transparent border?

Reply:
Right now the only places that the UX team knows that they want these indicators is the buttons within the side navigation (per service) & the notification bell icon within the header.  I'm sure some other team will find another location where it may be valid, but we aren't sure yet.

Reply:
I was thinking maybe we can even have it as a util class and users could add it on any component they like.

Reply:
One component I was thinking I could see a team wanting to use this within is the tabs. So you may be right.

Reply:
Chatted with @mayank99 about this a little bit, we are going to only focus on the situations where we know this will be used.  It may be used in new locations in the future that will require some changes (like placed with a blue / green button).